lightning/src/events/mod.rs

```diff
@@ -603,6 +603,9 @@ pub enum Event {
 		/// If this is `true`, the forwarded HTLC was claimed by our counterparty via an on-chain
 		/// transaction.
 		claim_from_onchain_tx: bool,
+		/// The final amount forwarded, in milli-satoshis, after the fee is deducted.
+		/// The caveat described above the `fee_earned_msat` field applies here as well.
+		amount_forwarded_msat: Option<u64>,
```

Comment:
maybe `outgoing_amount_forwarded_msat`? 🤷

Reply:
Going with a blend of the above: `outbound_amount_forwarded_msat` (because `HTLCIntercepted`, also uses the word "outbound"...I do see "outgoing" is used elsewhere but at a glance not in anything exposed)